<ui version="4.0" >
 <author>Matthew Livingstone</author>
 <class>Form</class>
 <widget class="QWidget" name="Form" >
  <property name="windowModality" >
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <widget class="QGroupBox" name="groupBox" >
       <property name="title" >
        <string>File Details</string>
       </property>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>Separator:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="Separator" >
            <property name="text" >
             <string>,</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="label_5" >
          <property name="text" >
           <string>Special Characters:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="label_6" >
          <property name="text" >
           <string>Use \t to represent a TAB</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QPushButton" name="previewButton" >
          <property name="enabled" >
           <bool>true</bool>
          </property>
          <property name="text" >
           <string>Preview</string>
          </property>
          <property name="checkable" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0" >
         <layout class="QHBoxLayout" name="horizontalLayout" >
          <item>
           <widget class="QLabel" name="label_7" >
            <property name="text" >
             <string>No Data Value:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="NoDataValue" />
          </item>
         </layout>
        </item>
       </layout>
       <zorder>label_5</zorder>
       <zorder>label_6</zorder>
       <zorder>previewButton</zorder>
       <zorder>label_7</zorder>
       <zorder>NoDataValue</zorder>
       <zorder></zorder>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QTabWidget" name="tabWidget" >
       <property name="tabShape" >
        <enum>QTabWidget::Triangular</enum>
       </property>
       <property name="currentIndex" >
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab" >
        <attribute name="title" >
         <string>Points specifications</string>
        </attribute>
        <layout class="QVBoxLayout" >
         <item>
          <layout class="QGridLayout" >
           <item row="0" column="0" >
            <widget class="QLabel" name="label_8" >
             <property name="text" >
              <string>Choose Position Columns :</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" >
            <layout class="QGridLayout" >
             <item row="0" column="0" >
              <widget class="QLabel" name="label_2" >
               <property name="text" >
                <string>X</string>
               </property>
               <property name="alignment" >
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="1" >
              <widget class="QComboBox" name="x1" />
             </item>
            </layout>
           </item>
           <item row="2" column="0" >
            <layout class="QGridLayout" >
             <item row="0" column="0" >
              <widget class="QLabel" name="label_3" >
               <property name="text" >
                <string>Y</string>
               </property>
               <property name="alignment" >
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="1" >
              <widget class="QComboBox" name="y1" />
             </item>
            </layout>
           </item>
           <item row="3" column="0" >
            <layout class="QGridLayout" >
             <item row="0" column="0" >
              <widget class="QLabel" name="label_4" >
               <property name="text" >
                <string>Z</string>
               </property>
               <property name="alignment" >
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="1" >
              <widget class="QComboBox" name="z1" />
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2" >
        <attribute name="title" >
         <string>Properties</string>
        </attribute>
        <layout class="QVBoxLayout" >
         <item>
          <widget class="QTreeWidget" name="PropertiesTree" >
           <column>
            <property name="text" >
             <string>Properties</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QTableWidget" name="previewTable" />
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="pqUI.qrc" />
 </resources>
 <connections/>
</ui>
